datastream-server/src/main/java/com/linkedin/datastream/server/zk/ZkAdapter.java

```diff
@@ -848,8 +848,7 @@ public void cleanupOldUnusedTasks(Map<String, Set<DatastreamTask>> previousAssig
         previousAssignmentByInstance.values().stream().flatMap(Collection::stream).collect(Collectors.toSet());
     List<DatastreamTask> unusedTasks =
         oldTasks.stream().filter(x -> !newTasks.contains(x)).collect(Collectors.toList());
-    LOG.warn("Deleting the unused tasks {} found between previous {} and new assignment {}. ", unusedTasks,
-        previousAssignmentByInstance, newAssignmentsByInstance);
+    LOG.info("Deleting the unused tasks {} ");
```

Comment:
nit: it looks like you want to print a value, but you aren't passing anything to print in the log. Also why change it to warn?

Reply:
I will add unusedTask here. This is regular cleaning up process in every assignment, the log level should be info instead of warn